index.html

```diff
@@ -40,5 +44,202 @@ <h1>I´m Josué Glad to see you!</h1>
+            <li class="num">2015</li>
+        </ul>
+    </div>
+    <div class="text">
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <form class="buttom">
+        <button type="button" class="html">html</button>
+        <button type="button" class="css">css</button>
+        <button type="button" class="javaScript">javaScript</button>
+    </form>
+    <div>
+        <input type="button" name="boton1" value="See Project">
+    </div>
+    </header>
+    </section>
+
+
+<!--sixth part-->
+
+<section class="container-6">
+    <div class="title">
+        <h1 class="about">About Myself</h1>
+    </div>
+    <div class="description-test">
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+    <div class="lets-connect">
+        LET´S CONNECT
+    </div>
+    <div class="icons">
+        <i class="fa-brands fa-twitter"></i>
+        <i class="fa-brands fa-google"></i>
+        <i class="fa-brands fa-facebook"></i>
+        <i class="fa-brands fa-whatsapp"></i>
+        <i class="fa-brands fa-github"></i>
+        <i class="fa-brands fa-instagram"></i>
+        <i class="fa-brands fa-linkedin-in"></i>
+    </div>
+    <div>
+        <input type="button" name="boton1" value="Get my resume">
+    </div>
+</section>
+
+<!--Seventh part-->
+
+<section class="conatiner-7">
+    <div class="languajes">
+     <h2>Languajes
+    <i class="fa-solid fa-angle-down"></i>
+    </h2>
+    </div>
+    <div class="two-list">
+        <ul class="works">
+            <li class="java"><img src="images/Ellipse1.png" alt="js" class="js">JavaScript</li>
+            <li class="ht"><img src="images/Ellipse2.png" alt="Html" class="ht">HTML</li>
+            <li class="cs"><img src="images/Ellipse3.png" alt="css" class="cs">CSS</li>
+        </ul>
+    </div>
+    <div class="frameworks">
+        Frameworks
+        <i class="fa-solid fa-angle-right"></i>
+    </div>
+    <div>
+        <hr>
+    </div>
+    <div class="skills">
+        Skills
+        <i class="fa-solid fa-angle-right"></i>
+        <div>
+            <hr>
+        </div>
+    </div>
+</section>
+<section class="form-section">
+    <div class="form-bg2"></div>
+    <div class="form">
+        <h1>Contact me</h1>
+        <p>If you have an application you are interested in developing, a feature 
+            that you need built or a project that needs coding. I’d love to help with it </p>
+            <input type="text" id="form-name" placeholder="Yeremias">
+            <input type="email" id="form-email" placeholder="natajayanj14@gmail.com">
+            <input type="text" id="form-message" placeholder="Write your message here">
+        <div class="form-button">
+            <button>Get It Touch</button>
```

Comment:
None of your buttons are following the interaction styling from the template design. I'm marking just this one for brevity but you should fix all of them, not just the one in this section.

<img width="727" alt="Screen Shot 2022-06-08 at 13 02 28" src="https://user-images.githubusercontent.com/82426682/172685881-a79c46dd-b760-4ef7-bb39-ddb78180f206.png">

<img width="908" alt="Screen Shot 2022-06-08 at 13 02 43" src="https://user-images.githubusercontent.com/82426682/172685918-fd157691-5465-4e4c-9e69-a1894f400f74.png">